Packs/Troubleshoot/Scripts/TroubleshootTestInstance/TroubleshootTestInstance.py

```diff
@@ -74,11 +93,15 @@ def main():
             }
         }
         if errors:
-            err_str = "\n".join(errors)
+            err_str = '\n'.join(errors[0].splitlines()[:10])
             human_readable = f'Found errors in instance {instance_name} after changing the next keys: ' \
                              f'{", ".join(keys_in_instance)}\n' \
-                             f'Errors:\n{err_str}'
-            return_error(human_readable, outputs=context)
+                             f'First lines of the error:\n{err_str}'
+            demisto.results(fileResult(
+                'test-module-results.txt',
```

Comment:
can we add a test for this?

Reply:
for what?

Reply:
to see we get the result file as expected